index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="css/styles.css" rel="stylesheet" type="text/css">
+	<title>Portfolio - Ruth Abreu</title>
+</head>
+<body>
+	<header class="app-bar">
+		<div class="toolbar">
+			<div class="image-container" id="logo">
+				<a href="index.html"><img src="images/Welcome.svg" alt="Logo" /></a>
+			</div>
+			<nav class="image-container" id="hamburger">
+				<img src="images/Hamburguer-menu.svg" alt="Menu" />
+			</nav>
+		</div>
+	</header>
+	<main>
+		<section id="headline">
+			<div class="primary">
+				<h1 id="i-am">Hey there. I’m Ruth</h1>
+				<h1 id="i-am-a">I’m a software developer</h1>
```

Comment:
- [ ] Nice work here. However, I advise you to use just one `h1` tag and `hx` for the other. you can also use the second text "I am a software developer" on the same line as the first, but using a `br` tag to keep them on two lines.

- In HTML, the h1 element is typically used as the top-level heading on a page or section of content, and it is recommended to use only one h1 element per page or section.

-This is because the h1 element serves an important role in structuring the content and signaling to search engines, screen readers, and other tools which content is the main heading and what the page or section is about. Using multiple h1 elements on a page or section can create confusion and ambiguity about the hierarchy of the content, which can negatively impact search engine rankings and accessibility.

-Instead, it is recommended to use the `h2` element for subheadings that are of a lower level of importance than the main h1 heading, and to use the h3, h4, and so on for additional levels of subheadings. This creates a clear hierarchy of headings that can help organize the content and make it easier to read and understand.